src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PractitionerMapper.java

```diff
@@ -0,0 +1,27 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
+
+import java.util.Optional;
+import org.hl7.fhir.dstu3.model.Practitioner;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.RequesterNameAndAddress;
+
+@Component
+public class PractitionerMapper {
+    protected static final String SDS_USER_SYSTEM = "https://fhir.nhs.uk/Id/sds-user-id";
```

Comment:
`private` rather than `protected`

Reply:
It's `protected` so we can use it directly in the test. Happy to restrict this and copy the value for the test.

Reply:
Not sure what other people do, but I prefer to copy the value to the test, so if this constant ever changes, then the test itself will fail (until someone updates it there)